From 7762946cda68bb8f616924592246bcda2da746c9 Mon Sep 17 00:00:00 2001
From: Stefan Christ <contact@stefanchrist.eu>
Date: Wed, 5 Oct 2016 20:34:14 +0200
Subject: [PATCH 4322/4706] drm/fb-helper: fix sphinx markup for
 DRM_FB_HELPER_DEFAULT_OPS

commit 21bf75eca03c9da000cd7368c6e8eec7559debe5 upstream

Fix invalid sphinx markup in the comment for the newly added
DRM_FB_HELPER_DEFAULT_OPS.

Signed-off-by: Stefan Christ <contact@stefanchrist.eu>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Link: http://patchwork.freedesktop.org/patch/msgid/1475692454-11543-1-git-send-email-contact@stefanchrist.eu
---
 include/drm/drm_fb_helper.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/drm/drm_fb_helper.h b/include/drm/drm_fb_helper.h
index 3c5f599..ed8edfe 100644
--- a/include/drm/drm_fb_helper.h
+++ b/include/drm/drm_fb_helper.h
@@ -218,7 +218,7 @@ struct drm_fb_helper {
 };
 
 /**
- * @DRM_FB_HELPER_DEFAULT_OPS:
+ * define DRM_FB_HELPER_DEFAULT_OPS - helper define for drm drivers
  *
  * Helper define to register default implementations of drm_fb_helper
  * functions. To be used in struct fb_ops of drm drivers.
-- 
1.7.5.4

